src/store/zap/actions.js

```diff
@@ -833,8 +833,10 @@ export function updateSelectedUcComponentState(context, projectInfo) {
   })
 }
 
-export function setDirtyState(context, isDirty) {
-  context.commit('setDirtyState', isDirty)
+export function setDirtyState(context) {
```

Comment:
This makes little sense.... Why are we putting dirty state into the store?

Reply:
this was previously written code that sends the dirty state value in the store to Studio V6

the value was never being updated 

this PR adds a REST_API which updates it